formats/application/vnd/oada/isoblue/can/1/+json/schema.js

```diff
@@ -0,0 +1,68 @@
+/* Example:
```

Comment:
This example should go in examples/default.js rather than as a comment at the top of the schema file